frontend/workflows/projectCatalog/src/details/card.tsx

```diff
@@ -0,0 +1,149 @@
+import React from "react";
+import { Card, ClutchError, Error, Grid, styled, Typography } from "@clutch-sh/core";
+import { LinearProgress } from "@material-ui/core";
+
+export type CardTypes = "Dynamic" | "Metadata";
+
+export interface DetailsCard {
```

Comment:
If you can use meta and dynamic card types below you can remove the export here

Reply:
Yeah this was only so that it could be extended in custom cases, if we dont think we need that I can remove it though.